### PR TITLE
Docs improvements

### PR DESCRIPTION
**Pull request by bwohlberg**
Various docs fixes and improvements.

### CODE REVIEW

File: docs/source/include/operator.rst [hunk 1]
```diff
@@ -111,17 +110,17 @@ Using A LinearOperator
 We implement two ways to evaluate a :class:`.LinearOperator`. The first is using standard
 callable syntax: ``A(x)``. The second mimics the NumPy matrix multiplication
 syntax: ``A @ x``. Both methods perform shape and type checks to validate the
-input before ultimately either calling `A._eval` or generating a new :class:`.LinearOperator`.
+input before ultimately either calling :code:`A._eval` or generating a new :class:`.LinearOperator`.
```
Comment (Michael-T-McCann) [inline docs/source/include/operator.rst:113]: consider adding a note about ":code:" to https://scico.readthedocs.io/en/stable/style.html#markup

File: docs/source/include/optimizer.rst [hunk 2]
```diff
@@ -47,14 +48,17 @@ in :eq:`eq:admm_prob`, corresponding to defining
...
+problem, and the :math:`g_i` are :class:`.Functional`, typically
+corresponding to a regularization term or constraint. Each of the
+:math:`g_i` must have a proximal operator defined. It is also possible
+to set ``f = None``, which corresponds to defining :math:`f = 0`,
+i.e. the zero function.
 
 
-Subproblem Solvers
-^^^^^^^^^^^^^^^^^^
+Sub-Problem Solvers
```
Comment (Michael-T-McCann) [inline docs/source/include/optimizer.rst:60]: Chicago manual of style disagrees, https://www.chicagomanualofstyle.org/book/ed17/part2/ch07/psec089.html Section 4.

**Review by Michael-T-McCann — COMMENTED** [review]
Couple suggestions

File: docs/source/include/optimizer.rst (hunk 2 above)
Comment (bwohlberg) [reply docs/source/include/optimizer.rst]: Fixed.

**Pull request merged**